authors/apps/authentication/views.py

```diff
@@ -18,12 +18,14 @@ class RegistrationAPIView(GenericAPIView):
 
     def post(self, request):
         user = request.data.get('user', {})
+        # Calling the validate user registration data before passing it to the serializer to save it
+
         # The create serializer, validate serializer, save serializer pattern
         # below is common and you will see it a lot throughout this course and
         # your own work later on. Get familiar with it.
         serializer = self.serializer_class(data=user)
-        serializer.is_valid(raise_exception=True)
-        serializer.save()
+        if serializer.is_valid(raise_exception=True):
```

Comment:
@felixkiryowa please undo the chnages to this file as well

Reply:
I have fixed it